network-mux/src/Network/Mux/TCPInfo.hsc

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE CPP                 #-}
+
+#ifdef linux_HOST_OS
```

Comment:
Instead of a top level #ifdef we can conditionally include this module in `other-modules`.

Reply:
That file defines a `StructTCPInfo` regardless of OS which is used when defining the `MuxTraceTCPInfo`.

Reply:
I've changed to a conditional inclusion of TCPInfo in the cabal file and updated the definition of MuxTraceTCPInfo